src/test/java/net/snowflake/ingest/streaming/internal/ParquetValueParserTest.java

```diff
@@ -14,6 +14,9 @@
 
 public class ParquetValueParserTest {
 
+  public static final float DEFINITION_LEVEL_ENCODING_BYTES = 1 + 1.0f / 8;
+  private static final int BYTE_ARRAY_LENGTH_ENCODING_BYTES = 4;
```

Comment:
I would put these statics and also use in the `ParquetValueParser`:
```
public static final float BIT_ENCODING_BYTE_LEN = 1.0f / 8;
public static final float DEFINITION_LEVEL_ENCODING_BYTE_LEN = 2.0f / 8;
public static final int BYTE_ARRAY_LENGTH_ENCODING_BYTE_LEN = 4;
```

Reply:
done.